qlib/data/dataset/processor.py

```diff
@@ -106,6 +106,22 @@ def process_inf(df):
 
         return replace_inf(df)
 
+class Fillna(Processor):
+    """Process infinity  """
+
+    def __call__(self, df):
+        def fill_na(data):
+            def process_na(df):
+                for col in df.columns:
+                    # FIXME: Such behavior is very weird
+                    df[col] = df[col].fillna(0)
+                return df
+
+            data = datetime_groupby_apply(data, process_na)
```

Comment:
I'm not sure  whether `datetime_groupby_apply` will be faster than  fillna direclty.

We use  `datetime_groupby_apply` in ProcessInf because of it is compute-intensive.
Fillna is not.